modules/swagger-jaxrs/src/main/java/com/wordnik/swagger/jaxrs/Reader.java

```diff
@@ -710,6 +735,31 @@ private static boolean isResourceClass(Class<?> cls) {
     return cls.getAnnotation(Api.class) != null;
   }
 
+  private List<Parameter> collectGlobalParameters(Class<?> cls) {
```

Comment:
ok - that could also be moved out perhaps - but lets not touch these for now... thanks